cmd/kubeadm/app/phases/upgrade/postupgrade_test.go

```diff
@@ -114,26 +113,23 @@ func TestRollbackFiles(t *testing.T) {
 }
 
 func TestWriteKubeletConfigFiles(t *testing.T) {
-	// exit early if the user doesn't have root permission as the test needs to create /etc/kubernetes directory
-	// while the permission should be granted to the user.
-	isPrivileged := preflight.IsPrivilegedUserCheck{}
-	if _, err := isPrivileged.Check(); err != nil {
-		return
+	tempDir, err := ioutil.TempDir("", "test-kubelet-config-files")
```

Comment:
missed this.
we should use os.TempDir because ioutil.TempDir is deprecated.
i updated my example after posting it.
https://pkg.go.dev/io/ioutil#TempDir

Reply:
sure, shall update it !

Reply:
Or create a tmpdir using t.TempDir() which gets auto-cleaned

Reply:
Thanks @liggitt . I tested with `t.TempDir()` and it's pretty much serving the purpose. Shall be updating the PR.